README.md

```diff
@@ -105,6 +105,64 @@ const tracer = opentracing.globalTracer();
 
 Note: `globalTracer()` returns a wrapper on the actual tracer object. This is done for the convenience of use as it ensures that the function will always return a non-null object.  This can be helpful in cases where it is difficult or impossible to know precisely when `initGlobalTracer` is called (for example, when writing a utility library that does not control the initialization process).  For more precise control, individual `Tracer` objects can be used instead of the global tracer.
 
+### Span manager (experimental)
+
+This library supports span activation in the spirit of the [Scope Manager](https://github.com/opentracing/specification/blob/f7ca62c9/rfc/scope_manager.md) draft RFC.
+
+If neither `childOf` nor `references` is specified, create spans are children of the activated span.
+
+The intent of `SpanManager` is that it remains active within the executed function and its transitive call graph. Ideally, this applies to both synchronous and asynchronous calls.
+
+```js
+tracer.activeSpan() // null
+tracer.activate(span, () => {
+    tracer.activeSpan() // span
+    setTimeout(() => {
+        tracer.activeSpan() // span
+    }, 0);
+    (async () => {
+        tracer.activeSpan() // span
+    })();
+});
+tracer.activeSpan() // null
+```
+
+JavaScript support for continuation local storage is a patchwork.
+This library includes simple wrappers around a few common APIs for continuation contexts.
```

Comment:
This should be selected automatically for the user according to the environment. This will avoid the user needing extensive knowledge of all the possible ways to do context propagation in JavaScript.

I've mentioned how I think this should be selected at the bottom of https://github.com/opentracing/opentracing-javascript/issues/112#issuecomment-433982906